ethcore/builtin/src/lib.rs

```diff
@@ -600,8 +685,117 @@ mod tests {
 	use num::{BigUint, Zero, One};
 	use parity_bytes::BytesRef;
 	use rustc_hex::FromHex;
+	use hex_literal::hex; // todo[dvdplm] switch tests to hex_literal
```

Comment:
well, `hex_literal` is used in the tests in this PR.

I guess it refers to the other tests right? Maybe clarify or create an issue for it

Reply:
I'll port over all the other tests to use `hex!`. I am so fed up with `rustc-hex`. :)